docs/data/rpc/api-reference/methods/getLedgerEntries.mdx

```diff
@@ -48,291 +147,179 @@ print(
 )
 ```
 
-#### JavaScript
-
-If you are using the [JavaScript](https://stellar.github.io/js-stellar-sdk/) `stellar-sdk` to generate these keys, you will need to install the latest pre-release version of the SDK. This can be done like so:
-
-```bash
-yarn add @stellar/stellar-sdk
-```
-
 ```js
 import { xdr, Address } from "@stellar/stellar-sdk";
 
 const getLedgerKeySymbol = (contractId, symbolText) => {
-  const ledgerKey = xdr.LedgerKey.contractData(
+  return xdr.LedgerKey.contractData(
     new xdr.LedgerKeyContractData({
       contract: new Address(contractId).toScAddress(),
       key: xdr.ScVal.scvSymbol(symbolText),
+      // or, possibly, .temporary(), depending on your contract
```

Comment:
the [increment contract example](https://developers-pr1060.previews.kube001.services.stellar-ops.com/docs/build/smart-contracts/getting-started/storing-data) stores this variable in instance storage. So this comment might be confusing. Maybe you can rephrase the comment to be 
```suggestion
      // the increment contract stores its state in persistent storage but other contracts may use
      // temporary storage which have a durability of xdr.ContractDataDurability. temporary()
```